samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/MessageBot.cs

```diff
@@ -153,11 +191,16 @@ protected override async Task<TaskModuleResponse> OnTeamsTaskModuleSubmitAsync(I
                             }
                             break;
                         case "transfer":
+                            Identity transferee = new Identity { Id = turnContext.Activity.From.AadObjectId };
```

Comment:
I figured I'd move the setting of the tenant away from callService where we used the value defined in settings to setting it from turncontext here instead.